nncf/experimental/openvino_native/quantization/quantize.py

```diff
@@ -11,79 +11,106 @@
-                                   validation_fn: Callable[[Any, Iterable[Any]], float],
-                                   max_drop: float = 0.01,
-                                   preset: QuantizationPreset = QuantizationPreset.PERFORMANCE,
-                                   target_device: TargetDevice = TargetDevice.ANY,
-                                   subset_size: int = 300,
-                                   fast_bias_correction: bool = True,
-                                   model_type: Optional[ModelType] = None,
-                                   ignored_scope: Optional[IgnoredScope] = None,
-                                   max_num_iterations: int = sys.maxsize) -> ov.Model:
+@tracked_function(NNCF_OV_CATEGORY,
+                  [CompressionStartedWithQuantizeApi(),
+                   "target_device", "preset", "max_drop"])
+def quantize_with_accuracy_control_impl(
+    model: ov.Model,
+    calibration_dataset: Dataset,
+    validation_dataset: Dataset,
+    validation_fn: Callable[[Any, Iterable[Any]], float],
+    max_drop: float = 0.01,
+    preset: QuantizationPreset = QuantizationPreset.PERFORMANCE,
+    target_device: TargetDevice = TargetDevice.ANY,
+    subset_size: int = 300,
+    fast_bias_correction: bool = True,
+    model_type: Optional[ModelType] = None,
+    ignored_scope: Optional[IgnoredScope] = None,
+    advanced_quantization_parameters: Optional[AdvancedQuantizationParameters] = None,
+    advanced_accuracy_restorer_parameters: Optional[AdvancedAccuracyRestorerParameters] = None) -> ov.Model:
     """
     Implementation of the `quantize_with_accuracy_control()` method for the OpenVINO backend via the
     OpenVINO Runtime API.
     """
-    quantized_model = quantize_impl(model, calibration_dataset, preset, target_device, subset_size,
-                                    fast_bias_correction, model_type, ignored_scope, compress_weights=False)
+    if advanced_accuracy_restorer_parameters is None:
+        advanced_accuracy_restorer_parameters = AdvancedAccuracyRestorerParameters()
+    
+    if advanced_accuracy_restorer_parameters.tune_hyperparams:
+        raise RuntimeError('Quantization algorithm with accuracy control from the '
+                           'OpenVINO backend does not support tuning hyperparams yet')
+    if advanced_accuracy_restorer_parameters.convert_to_mixed_preset:
+        raise RuntimeError('Quantization algorithm with accuracy control from the '
+                           'OpenVINO backend does not support the option to convert '
+                           'to the mixed preset yet')
+    
+    compress_weights = is_weight_compression_needed(advanced_quantization_parameters)
+    
+    if advanced_quantization_parameters is None:
+        copied_parameters = AdvancedQuantizationParameters()
+    else:
+        copied_parameters = deepcopy(advanced_quantization_parameters)
+    copied_parameters.backend_params[BackendParameters.COMPRESS_WEIGHTS] = False
```

Comment:
I set `False` because the default quantization should be called with `compress_weights = False`. https://github.com/openvinotoolkit/nncf/blob/7f09a5d9c5678f22cab35680133fcecf67296401/nncf/experimental/openvino_native/quantization/quantize.py#L111